app/styles/ui/_branches.scss

```diff
@@ -57,6 +57,12 @@
     padding-bottom: var(--spacing-half);
   }
 
+  .selected:focus {
```

Comment:
Can we make this `.list-item.selected:focus`? Not that it matters but I had to think for a second about why `.selected` was nested under `.pull-request-list`